src/v/cluster/members_manager.h

```diff
@@ -51,6 +55,10 @@ class members_manager {
     ss::future<result<configuration_update_reply>>
       handle_configuration_update_request(configuration_update_request);
 
+    bool is_batch_applicable(const model::record_batch& b) {
+        return b.header().type == model::record_batch_type::node_management_cmd;
+    }
```

Comment:
I'm confused because the previous commit removes this and says its not used

Reply:
it is not used since it isn't wired in stm yet,

Reply:
it handles config batches not via STM but through raft notification

Reply:
raft enables configuration before it is applied to state

Reply:
I have a plan to move node management out of Raft configuration.

Reply:
I see. then perhaps the code for handling raft::group_configuration should be moved to the patch that removes that support? or combine them into a single commit and write a single sentence in the commit that explains what's happening...

Reply:
The code handling `raft::group_configuration` didn't change. This method is implemented but never called.